webapp/apps/taxbrain/views.py

```diff
@@ -766,9 +761,9 @@ def get_result_context(model, request, url):
 
     if (model.json_text is not None and (model.json_text.raw_reform_text or
        model.json_text.raw_assumption_text)):
-        reform_file_contents = model.json_text.reform_text
+        reform_file_contents = model.json_text.raw_reform_text
         reform_file_contents = reform_file_contents.replace(" ","&nbsp;")
-        assump_file_contents = model.json_text.assumption_text
+        assump_file_contents = model.json_text.raw_assumption_text
         assump_file_contents = assump_file_contents.replace(" ","&nbsp;")
```

Comment:
After #641, all reforms have attribute `assumption_text` as an assumption dictionary but the values are empty if they are not specified via an assumptions file or the dynamic behavior interface.  So, `raw_assumption_text` was used instead to distinguish file-upload reforms with an assumptions file from other reforms.